File: CursoEmVideo/POO [60 Horas]/PHP POO [40 Horas]/Aula 11/Tecnico.php
<?php
    require_once '..//Aula 10//Aluno.php';
    class Tecnico extends Aluno {
        private $registroProfissional;

        function getRegistroProfissional() {
            return $this -> registroProfissional;
        }

        function setRegistroProfissional($registroProfissional) {
            $this -> registroProfissional = $registroProfissional;
        }

        function praticar() {
            print("Estou praticando."."<br>");
        }
    }

File: CursoEmVideo/POO [60 Horas]/Java POO [40 Horas]/Aula 11/Tecnico.java
public class Tecnico extends Aluno {
    private String registroProfissional;

    public String getRegistroProfissional() {
        return registroProfissional;
    }

    public void setRegistroProfissional(String registroProfissional) {
        this.registroProfissional = registroProfissional;
    }

    public void praticar() {
        System.out.println("Estou praticando.");
    }
}
